File: C assignments/basic c prog/swap.c
#include<stdio.h>
int main()
{
    int a,b;
    printf("Numbers before Swapping a= ");
    scanf("%d",&a);
    printf("b= ");
    scanf("%d",&b);
    int c=a;
    a=b;
    b=c;
    printf("numbers after swapping a=%d b=%d ",a,b);

    
}

File: COP Java/day3/src/ASSIGNMENT/Swap.java
package ASSIGNMENT;
import java.util.Scanner;
public class Swap {

	public static void main(String[] args) {
		int a,b,temp;
		Scanner sc ;
		sc = new Scanner(System.in);
		System.out.println("enter numbers a and b to swap");
		
		a = sc.nextInt();
		b = sc.nextInt();
		temp = a;
		a = b;
		b = temp;
		System.out.println("after swapping a:"+a+"b="+b);
		
		

	}

}
